x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/description_step/helpers.tsx

```diff
@@ -218,21 +229,37 @@ export const buildUrlsDescription = (label: string, values: string[]): ListItems
       {
         title: label,
         description: (
-          <EuiFlexGroup gutterSize="none" alignItems="flexStart" direction="column">
-            {values.map((val: string) => (
-              <EuiFlexItem>
-                <ReferenceLinkItem
-                  href={val}
-                  target="_blank"
-                  iconType="link"
-                  size="xs"
-                  flush="left"
-                >
-                  {val}
-                </ReferenceLinkItem>
-              </EuiFlexItem>
-            ))}
-          </EuiFlexGroup>
+          <EuiText size="s">
+            <ul>
+              {values
+                .filter(v => !isEmpty(v))
+                .map((val: string, index: number) => (
+                  <li data-test-subj="urlsDescriptionReferenceLinkItem" key={`${index}-${val}`}>
+                    <EuiLink href={val} external target="_blank">
+                      {val}
+                    </EuiLink>
+                  </li>
+                ))}
+            </ul>
+          </EuiText>
+        ),
+      },
+    ];
+  }
+  return [];
+};
+
+export const buildNoteDescription = (label: string, note: string): ListItems[] => {
+  if (note) {
```

Comment:
Nit: I would usually add a `if (note.trim() == '')` to account for empty spaces but maybe you don't want to.

Pointing out this is a bit misleading as although it will do what you want it made me think a bit more if you're trying to check if note is an empty string vs a null/undefined. I am assuming it is for an empty string?

No changes asked, just pointing out as a maintainer I don't really know.

In the unit tests you don't exercise this branching logic either which would make it easier for someone to make a false assumption and remove it and not see any tests break if you are ensuring that `note` is not an empty string there and want to return an empty array if it is an empty string.

Reply:
Good point - thanks! Same as above, I'll plan to make this change in the upcoming PR.

Reply:
Ended up adding this to the PR as had to update circular dep check file.